### DISCUSSION TITLE
Pivot2PCAP Issue (2.3.130)

### DISCUSSION THREAD

**Discussion by xfaith** [Security-Onion-Solutions/securityonion#8159] [Unsupported Versions] [answered]
I was playing around today and tried to pivot2pcap from an alert panel.  

![image](https://user-images.githubusercontent.com/1995708/174671641-692d9bbc-98d9-4a00-8b52-c68a4c12a143.png)
This is the error I am receiving currently.  

so-status  all [OK]
I did:
so-pcap-restart  [No change]
so-soc-restart [no change]
reboot - [No change]

Logs
Stenographer:  2022/06/20 20:47:14 http: TLS handshake error from 127.0.0.1:36560: EOF

Sensoroni: 
![image](https://user-images.githubusercontent.com/1995708/174676541-e8ae6cfc-a984-463f-8a4f-e0d9fb633d07.png)

This is just a home setup for testing/playing.  
SecurityOnion Setup
VM Stand-alone 2.3.130
12G Mem
4 CPU

Not sure where else to look

**Comment by dougburks** [comment]
What was the date on the alert and does your pcap storage go back that far?

You can also try manually querying Stenographer for the pcap as shown here:
https://docs.securityonion.net/en/2.3/stenographer.html#command-line

**Comment by xfaith** [comment, marked as answer]
So I cleared everything out, after i noticed that /nsm was at 90%.  Did a so-elastic-clear , and rm (/nsm/pcap && /nsm/pcapindex)  Didnt seem to do anynting.  

Then I tried to the querying as suggested, which looks like its a CERT issues, which reminded me of the time I was playing with Max exporting PCAP and was trying to figure out other ways, which I may have tried to copy the certs/and steno docker, so I probably did some other things and broke it.  

O well, not a big issue really.  

here is the error if you think its reversible:" 
![image](https://user-images.githubusercontent.com/1995708/174965410-5dcde494-cf40-4cdf-9970-0b9b5e5f80ea.png)

**Reply by dougburks** [reply]
> Then I tried to the querying as suggested, which looks like its a CERT issues, which reminded me of the time I was playing with Max exporting PCAP and was trying to figure out other ways, which I may have tried to copy the certs/and steno docker, so I probably did some other things and broke it.

Yes, that sounds like the root issue.
